core/src/Cabana_HDF5ParticleOutput.hpp

```diff
@@ -31,6 +31,11 @@
 #include <Kokkos_Core.hpp>
 
 #include <hdf5.h>
+#if H5_HAVE_SUBFILING_VFD
```

Comment:
These should probably all be #ifdefs instead of #ifs or else I believe it will generate compiler warnings when Subfiling isn't enabled.

Reply:
Fixed.